products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
So is it that or this related bit which is spitting out the error
4 /usr/local/bin/bundle:23:in `<main>'
13:33:24 Error cdn-ifying img <img src="#%7Bcdn(site.base_url%20+%20'/images/products/multipath/cpp-logo.png')%7D" alt="Eclipse CPP Logo" title="" developer toolset get started developing with c and eclipse on rhel>
13:33:24 While processing file /home/awestruct/developer.redhat.com/products/developertoolset/get-started-rhel6-cpp.adoc
13:33:24 An error occurred: Is a directory @ io_fread - /home/awestruct/developer.redhat.com